### ISSUE TITLE
btSoftBody missing methods and properties

### ISSUE THREAD

**Issue by microspud** [kripken/ammo.js#178]
Hi, 

We have just started working with soft bodies, however the majority of methods and properties are missing. Is it a case of just adding the missing methods and property bindings to the ammo.idl file and then rebuilding? Or is there more involved?
Many thanks..

**Comment by kripken** [comment]
Yes, it should just be a matter of adding to the idl file. Please open a PR with any additions.
